src/events/aws_sdk.js

```diff
@@ -230,6 +230,11 @@ const SQSEventCreator = {
             });
             break;
         case 'sendMessageBatch':
+            if (response.data.Successful && response.data.Successful.length > 0) {
+                eventInterface.addToMetadata(event, {
+                    record: JSON.stringify(response.data.Successful.map(item => item)),
```

Comment:
what happened if json.stringify throws an error?

Reply:
Internal exception that only fails the event

Reply:
where do you catch the error?